internal/graphql/resolver/addons.go

```diff
@@ -21,7 +22,7 @@ func (r *Resolver) getAvailableAddons(ctx context.Context, provider models.Provi
 		selectors = append(selectors, *selector)
 	}
 
-	addonlist, err := model.GetAddonsState(selectors, r.Config.KubeClient, provider)
+	addonlist, err := services.GetAddonsState(selectors, r.Config.KubeClient, provider)
```

Comment:
Why chance from `model` to `service` package name? We have been using `model` convention in Meshery.

Reply:
Since I need to decouple "model"-definition generated by gqlgen and the "models"-function like

Reply:
Why the need to decouple? The resolvers are both generated and hand written, and we still use the same package for it.

Reply:
For me, it doesn't make sense to put the type definitions for the resolvers to be in the same package as functions for talking to the database. The other thing is that it's difficult to connect the packages due to cyclic imports.